### DISCUSSION TITLE
docs/v1.10.x/guide/setup-options/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#1915] [Announcements]
# docs/v1.10.x/guide/setup-options/

https://kubeovn.github.io/docs/v1.10.x/guide/setup-options/

**Comment by tghfly** [comment]
在1.10.6版本里 ENABLE_KEEP_VM_IP 默认为true
ENABLE_KEEP_VM_IP=${ENABLE_KEEP_VM_IP:-true}

**Reply by oilbeater** [reply]
文档已更新，感谢发现问题
